README.md

```diff
@@ -95,12 +95,10 @@ Astro is free, open source software made possible by these wonderful sponsors.
 [❤️ Sponsor Astro! ❤️](https://github.com/withastro/.github/blob/main/FUNDING.md)
 
 <p align="center">
-  <a target="_blank" href="https://github.com/sponsors/withastro">
+  <a target="_blank" href="https://opencollective.com/astrodotbuild">
 
-[![Astro's sponsors.](https://astro.build/sponsors.png "Astro's sponsors.
-Platinum sponsors: Vercel, storyblok, Netlify, Ship Shape, Google Chrome
-Gold sponsors: ‹div›RIOTS, DEEPGRAM, Transloadit, CloudCannon
-Sponsors: Monogram, Qoddi, Dimension")](https://github.com/sponsors/withastro)
+[![Astro's sponsors.](https://astro.build/sponsors.png "Sponsor logos
+including the current Astro Sponsors, Gold Sponsors, and Exclusive Partner Sponsors: Netlify, Sentry, and Project IDX.")](https://github.com/sponsors/withastro)
```

Comment:
```suggestion
![Astro's sponsors.](https://astro.build/sponsors.png "Sponsor logos
including the current Astro Sponsors, Gold Sponsors, and Exclusive Partner Sponsors: Netlify, Sentry, and Project IDX.")
```
I think because we have the separate `<a>` tag with a link, we either *don't* need this link at all, or it should also change to the open collective one